website/content/api-docs/system/managed-keys.mdx

```diff
@@ -132,6 +132,44 @@ $ curl \
   a read-write session to the HSM. This is a boolean expressed as a string (e.g.
   `"true"`). This key is mainly to work around a limitation within AWS's CloudHSM v5
   pkcs11 implementation.
+  
+#### AWS KMS Parameters
+- `access_key` `(string: <required>)`: The AWS access key to use. This may also be provided in the
+`AWS_ACCESS_KEY_ID` environment variable or as part of the AWS profile from the AWS CLI or instance
+profile.
+
+- `curve` `(string: "P256")`: The curve to use for an ECDSA key. This field is used when
+`key_type` is `ECDSA`. Supported options are `P256`, `P384`, or `P521`. If a value is not specified,
+Vault defaults to `P256`.
+
+- `endpoint` `(string: "")`: Used to specify a custom AWS endpoint. If not set, Vault will use the
+default API endpoint for your region.
+
+- `key_bits` `(string: <required>)`: The size in bits for an RSA key. This field is required when
+`key_type` is `RSA`. Supported values are `2048`, `3072`, and `4096`.
+
+- `key_type` `(string: <required>)`: The type of key to use. Supported values are `RSA` and `ECDSA`
+(not case sensitive). If the key type is `RSA`, the `key_bits` field is also required. If the key
+type is `ECDSA`, the `curve` field is required.
+
+- `kms_key_id` `(string: <required>)`: An identifier for the key. If using an existing key, this
+value should be an AWS alias for the key, including the `alias/` prefix. If Vault should generate
+the key, the provided ID will be added as an alias after the key is created. To fully specify the
+alias for the generated key, the provided key ID should be the full alias, including the `alias/`
+prefix. Otherwise, Vault will create the alias as `alias/hashicorp/{kms_key_id}`. The alias must be
+unique with the AWS account.
+
+    e.g., If the value provided is `kms_key_id=example-key-id`, then Vault will create the alias
```

Comment:
Due to the indentation, this paragraph is going to appear as a code block:
<img width="882" alt="image" src="https://user-images.githubusercontent.com/14779/156612891-41f921c8-d6df-4015-8c9b-767678ea8ef6.png">

Perhaps it would be better to use `~>` to highlight it as a note, or to simply leave it as a regular paragraph?